Buildings/Experimental/DHC/Loads/Steam/BaseClasses/Examples/ValveSelfActing.mo

```diff
@@ -0,0 +1,85 @@
+  Buildings.Fluid.Sources.Boundary_pT sou(
+    redeclare package Medium = MediumSte,
+    use_p_in=true,
+    p(displayUnit="Pa"),
+    T(displayUnit="K") = MediumSte.saturationTemperature(sou.p),
+    nPorts=1) "Source"
+    annotation (Placement(transformation(extent={{-80,-10},{-60,10}})));
+  Buildings.Fluid.Sensors.SpecificEnthalpyTwoPort speEntIn(
+    redeclare package Medium = MediumSte,
+    m_flow_nominal=m_flow_nominal)
+    "Upstream specific enthalpy"
+    annotation (Placement(transformation(extent={{-50,-10},{-30,10}})));
+  Buildings.Fluid.Sensors.SpecificEnthalpyTwoPort speEntOut(
+    redeclare package Medium = MediumSte,
+    m_flow_nominal=m_flow_nominal)
+    "Downstream specific enthalpy"
+    annotation (Placement(transformation(extent={{10,-10},{30,10}})));
+  Modelica.Blocks.Noise.UniformNoise pInSig(
+    samplePeriod(displayUnit="s") = 1,
+    y_min=500000 + 100000,
+    y_max=500000 - 100000) "Noisy signal for inlet pressure"
+    annotation (Placement(transformation(extent={{-60,40},{-80,60}})));
+  Fluid.Sources.Boundary_pT sin(
+    redeclare package Medium = MediumSte,
+    p=250000,
+    T=MediumSte.saturationTemperature(sin.p),
+    nPorts=1) "Sink"
+    annotation (Placement(transformation(extent={{100,-10},{80,10}})));
+  Fluid.FixedResistances.PressureDrop res(
+    redeclare package Medium = MediumSte,
+    m_flow_nominal=m_flow_nominal,
+    dp_nominal=1000) "Pressure drop"
+    annotation (Placement(transformation(extent={{40,-10},{60,10}})));
+  inner Modelica.Blocks.Noise.GlobalSeed globalSeed
+    "Setting for sublibrary noise"
+    annotation (Placement(transformation(extent={{80,-60},{100,-40}})));
+equation
+  connect(sou.ports[1], speEntIn.port_a)
+    annotation (Line(points={{-60,0},{-50,0}}, color={0,127,255}));
+  connect(speEntIn.port_b, prv.port_a)
+    annotation (Line(points={{-30,0},{-20,0}},
+                                             color={0,127,255}));
+  connect(prv.port_b, speEntOut.port_a)
+    annotation (Line(points={{0,0},{10,0}},  color={0,127,255}));
+  connect(pInSig.y, sou.p_in) annotation (Line(points={{-81,50},{-90,50},{-90,8},
+          {-82,8}}, color={0,0,127}));
+  connect(sin.ports[1], res.port_b)
+    annotation (Line(points={{80,0},{60,0}}, color={0,127,255}));
+  connect(res.port_a, speEntOut.port_b)
+    annotation (Line(points={{40,0},{30,0}}, color={0,127,255}));
+  annotation (Icon(coordinateSystem(preserveAspectRatio=false)),
+    Diagram(coordinateSystem(preserveAspectRatio=false)),
+    __Dymola_Commands(file="modelica://Buildings/Resources/Scripts/Dymola/Experimental/DHC/Loads/Steam/BaseClasses/Examples/ValveSelfActing.mos"
+    "Simulate and plot"),
+    experiment(StopTime=15, Tolerance=1e-06),
+    Documentation(info="<html>
+<p>
+This example model demonstrates the performance of self-acting steam pressure regulator 
+with a noisy/varying inlet pressure signal and a ramped mass flow rate signal. The inlet 
```

Comment:
There is no ramp in the mass flow rate. You could actually add a varying pressure signal to the outlet boundary so that we would observe that the mass flow rate is dictated by the `dp` on the outlet.

Reply:
Sorry, this is a documentation mistake that I missed revising. Thanks for pointing it.
We have not used a ramped mass flow rate and by revising the inlet signal values (to fall below setpoint) we are able to observe the variation in mass flow rate.